src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -878,7 +878,15 @@ public void GetProcessesByName_ProcessNameMachineName_ReturnsExpected(string mac
         [PlatformSpecific(TestPlatforms.Windows)] // Accessing processes on remote machines is only supported on Windows.
         public void GetProcessesByName_RemoteMachineNameWindows_ReturnsExpected(string machineName)
         {
-            GetProcessesByName_ProcessNameMachineName_ReturnsExpected(machineName);
+            try
+            {
+                GetProcessesByName_ProcessNameMachineName_ReturnsExpected(machineName);
+            }
+            catch (InvalidOperationException)
+            {
+                // As we can't detect reliably if performance counters are enabled 
+                // we let possible InvalidOperationExceptions pass silently.
```

Comment:
Why can't we detect it reliably?  Have we followed up with an expert in Windows who could help?